src/backend/distributed/citus--6.0-1--6.0-2.sql

```diff
@@ -0,0 +1,9 @@
+/* citus--5.2-4--5.2-5.sql */
+
+CREATE FUNCTION shard_name(object_name regclass, shard_id bigint)
```

Comment:
This should probably be pg_catalog.shard_name. For historical reasons, our functions are all in pg_catalog.